docs/xailient_sdk_api.md

```diff
@@ -0,0 +1,86 @@
+# Xailient SDK API Documentation
+
+**Xailient SDK Version: 1.0.7**
+
+## dnn.Detector Class
+
+``` python
+detector = dnn.Detector(int)
+```
+
+#### Arguments
+
+| Arguments     | Input Type    | Default Value | Description   |
+| ------------- |:-------------:| :------------:| -------------|
+| cores _(optional)_         | int           | 1             |   Number of threads to start for running the Detector. This gives the ability to run multiple threads for inferencing. This value should be tuned and select based on the use case to maximise overall performance. |
```

Comment:
I suggest using "threads" instead of "cores" for the argument name since this parameter really controls how many threads can be used. How many threads to use is usually determined by how many cores the chip has, but that's just a heuristic.

Reply:
Got it.